wpa_supplicant/events.c

```diff
@@ -3674,6 +3674,9 @@ static void wpa_supplicant_event_disassoc_finish(struct wpa_supplicant *wpa_s,
 		if (wpas_p2p_4way_hs_failed(wpa_s) > 0)
 			return; /* P2P group removed */
 		wpas_auth_failed(wpa_s, "WRONG_KEY");
+#ifdef CONFIG_ZEPHYR
+		send_wifi_mgmt_conn_event(wpa_s, reason_code);
```

Comment:
Reason code here doesn't necessarily translate to WRONG_KEY, so, may be just send `WLAN_REASON_4WAY_HANDSHAKE_TIMEOUT`

Reply:
Okay.